core/.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
```

Comment:
Notice here that I'm using `@typescript-eslint/eslint-recommended` instead of `@typescript-eslint/recommended`. This extension only makes sure that typescript plays nicely with `eslint:recommended` but does not enable the typescript recommended rules. Currently, there are a gargantuan number of problems with `core` so doing it all once is a herculean effort. At the very least we're getting base safety.